main/webapp/modules/core/langs/translation-fr.json

```diff
@@ -790,7 +790,7 @@
     "core-about/contributors": "Contributeurs (par ordre chronologique)",
     "core-about/thanks": "Remerciements spéciaux",
     "core-about/history": "OpenRefine a été initialement développé sous le nom de «&nbsp;Freebase Gridworks&nbsp;» par <a href='$1'>$2</a>. Metaweb a été acquis par <a href='$3'>$4</a> en juillet 2010 et ils ont renommé le produit Google Refine. En octobre 2012, le produit a été renommé OpenRefine lors de sa transition vers un projet soutenu par la communauté.",
-    "core-about/definition": "OpenRefine est un outil puissant pour travailler avec des données désordonnées. Utilisez-le pour améliorer la cohérence des données, liez-le à des registres de données tels que Wikidata, augmentez-le avec des données provenant d'autres sources, transformez-les en différents formats pour que d'autres outils puissent les utiliser et contribuez-les aux sources d'origine. OpenRefine n'est pas un service Web mais une application de bureau qui s'exécute sur votre propre ordinateur, vous pouvez donc traiter les données sensibles en toute confidentialité.",
+    "core-about/definition": "OpenRefine est un outil puissant pour travailler avec des données désordonnées. Utilisez-le pour améliorer la cohérence des données, liez-le à des registres de données tels que Wikidata, complétez-le avec des données provenant d'autres sources, transformez-le en différents formats pour que d'autres outils puissent les utiliser et renvoyer les données à leurs sources. OpenRefine n'est pas un service Web, mais une application de bureau qui s'exécute sur votre propre ordinateur, vous pouvez donc traiter les données sensibles en toute en privé.",
```

Comment:
This is a complex sentence, and I am not sure if either the previous or newer versions are fully grammatically correct. A bit less literal translation may be good here? It is a bit unclear whether 'OpenRefine' is the object (liez-_le_) or 'les données' (in which case it should be liez-_le**s**_?) My French is very rusty, but I'd go in this direction:

`OpenRefine est un outil puissant pour travailler avec des données désordonnées. Utilisez OpenRefine pour améliorer la cohérence des données; liez vos données à des bases de connaissance tels que Wikidata, complétez-les avec des données provenant d'autres sources, transformez-les en différents formats pour que d'autres outils puissent les utiliser et renvoyer les données à leurs sources.` 

I am also wondering why `en toute confidentialité` is being replaced here - to make the sentence sound a bit simpler? Should it then not simply be `en privé` rather than `en toute en privé`?

`OpenRefine n'est pas un service Web, mais une application de bureau qui s'exécute sur votre propre ordinateur. Vous pouvez donc traiter les données sensibles en privé.`

@wetneb I think some feedback from a native speaker is welcome :-)

Reply:
Thanks for the review and those tips! I'll go the 'add via Weblate' route.